pkg/controllers/dynakube/extension/tls/conditions.go

```diff
@@ -0,0 +1,3 @@
+package tls
+
+const extensionsSelfSignedTLSSecretConditionType string = "ExtensionsSelfSignedTLSSecret"
```

Comment:
I think it would be better to have one condition type for both secrets . @StefanHauth wdyt?

Reply:
But then we can't see the status of each secret individually, right?

Reply:
Do you really need it? For me it sounds overkill, high level secret for both sounds good compromise.

Reply:
Because if we have two types, @albertogdd  should use both to check, and it is a bit painful and, again, overkill.

Reply:
When you say both secrets, you mean the self-signed TLS and the TLSRef one?
IMO the TLSRef one is created by the clients, so we shouldn't have a condition about it. That's why I just added one condition just for the self-signed TLS secret.

Reply:
Resolution: I will create a small ticket about the conditions (1 condition for the 2 secrets) and in this PR I will revert some changes and just fix the bug